clients/drcachesim/tools/reuse_distance.cpp

```diff
@@ -52,11 +52,12 @@ reuse_distance_t::reuse_distance_t(const reuse_distance_knobs_t &knobs)
     : knobs_(knobs)
     , line_size_bits_(compute_log2((int)knobs_.line_size))
 {
-    if (DEBUG_VERBOSE(2)) {
-        std::cerr << "cache line size " << knobs_.line_size << ", "
-                  << "reuse distance threshold " << knobs_.distance_threshold
-                  << ", distance limit " << knobs_.distance_limit << std::endl;
-    }
+    reuse_distance_t::knob_verbose = knobs.verbose;
+    IF_DEBUG_VERBOSE(2,
+                     std::cerr << "cache line size " << knobs_.line_size << ", "
+                               << "reuse distance threshold " << knobs_.distance_threshold
+                               << ", distance limit " << knobs_.distance_limit
+                               << std::endl);
```

Comment:
I would assume we don't really need to flush: this is std:cerr which is not doing buffering anyway.  I think all instances of `std::endl` in this file can be replaced with `\n` which will avoid needless calls to flush().

Reply:
Done.